src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1015,73 +1015,19 @@ protected boolean canBeInstantiated() {
 				}
 			}));
 
-		Classes.registerClass(new ClassInfo<>(PotionEffectType.class, "potioneffecttype")
-				.user("potion( ?effect)? ?types?") // "type" had to be made non-optional to prevent clashing with potion effects
+		Registry<PotionEffectType> petRegistry = BukkitUtils.getPotionEffectTypeRegistry();
+		if (petRegistry != null) {
```

Comment:
Everything seems fine, just concerned about `petRegistry` being misleading

Reply:
[P]otion[E]ffect[T]ype... and it's not used outside this class.
I just wanted to keep it short.
Really no reason for it to be anything else.